tests/Feature/Commands/TweetAboutWeeklySummaryCommandTest.php

```diff
@@ -29,7 +29,7 @@
 
     // Assert
     Twitter::assertTweetCount(1)
-        ->assertLastTweet("There were 2 streams last week. 👏 Thanks to all the streamers and viewers. 🙏🏻\n Find them here: ".route('archive'));
+        ->assertLastTweet("2 streams happened last week. 👏 Thanks to all the streamers and viewers. 🙏🏻\n Find them here: ".route('archive'));
```

Comment:
And please adapt the tests to check both variations of the text (singular/plural)

Reply:
New test added to check both instances!